cove_iati/lib/iati.py

```diff
@@ -11,7 +11,7 @@
 from cove.lib.exceptions import CoveInputDataError, UnrecognisedFileTypeXML
 
 
-def common_checks_context_iati(context, upload_dir, data_file, file_type, api=False):
+def common_checks_context_iati(context, upload_dir, data_file, file_type, api=False, openag=False, orgids=False):
```

Comment:
I think orgids should probably be on by default - I think we want to see it in the web interface.

Reply:
I agree that we'd like it in the web interface, but I don't want the additional overhead (UI design, etc) to slow down anything else in this sprint. So, if it's easy/cheap/free to add to the web UI, amazing, but if there's any real work involved, let's leave it and prioritise it against other work in non-sprint or our next IATI sprint.

Reply:
It won't be too difficult to add to the web interface, but I'd rather get this merged and leave that for another sprint or for the non-sprint board.  If we agree on that, then I think it should be off by default.

Reply:
Ah okay, makes sense.